src/rules/disallow-generators-in-describe-functions.js

```diff
@@ -4,15 +4,16 @@
  * Module dependencies.
  */
 
-import assert from 'assert';
+const assert = require('assert');
 
 /**
  * Export `disallowGeneratorsInDescribeFunctions`.
  */
 
-const disallowGeneratorsInDescribeFunctions = () => {};
+module.exports = function() {
+};
 
-disallowGeneratorsInDescribeFunctions.prototype = {
+module.exports.prototype = {
```

Comment:
While I agree that this looks very weird, every single rule is designed like on the official package. E.g. https://github.com/jscs-dev/node-jscs/blob/master/lib/rules/require-capitalized-constructors.js#L38

There are plans to [migrate the codebase to ES2015](https://github.com/jscs-dev/node-jscs/issues/1469), which implies rewriting all the rules, but I'm still open to change this now if you prefer.